tests/components/evohome/conftest.py

```diff
@@ -152,4 +154,7 @@ async def setup_evohome(
 
         assert mock_client.account_info is not None
 
-        return mock_client
+        try:
+            yield mock_client
+        finally:
+            await hass.data[DOMAIN]["coordinator"].async_shutdown()
```

Comment:
We don't like touching internals in tests.
Can you explain why you need to turn off the coordinator manually when it's already using `async_register_shutdown`?

Reply:
Ah. It 'should' really be a blind `except:`, and not a `finally:`.. 

It is to prevent an error being logged during teardown when the test failed. 

The idea being to limit non-useful logging during test/dev of tests.

Reply:
I will remove it.